### PR TITLE
Updated tutorial

### PR DESCRIPTION
**Pull request by ptersilie**
Updated and linked to the Tutorial.

When ok, I'll squash everything into a single commit and we can merge.

### CODE REVIEW

**Comment by ptersilie** [comment]
Best way to inspect the changes is through [this link](https://github.com/softdevteam/eco/blob/update_tutorial/tutorial/TUTORIAL.md)

File: tutorial/TUTORIAL.md [hunk 1]
```diff
@@ -23,32 +23,39 @@ Eco is a language aware editor. When creating a new file we need to specifiy in
...
+At the moment our program is still invalid as shown by the red exclamation mark (![](../lib/eco/gui/exclamation.png)) under *Parsing status* on the right-hand-side of the editor.
 Let's fix the program by closing the <html> tag.
 
 Type:
 ```
 </html>
 ```
-Eco attempts to parse the program on every keystroke. The parse tree can be
+![](2.png)
+
```
Comment (ltratt) [inline tutorial/TUTORIAL.md:44]: Shouldn't this screenshot go before `type </html>`?

File: tutorial/TUTORIAL.md [hunk 2]
```diff
@@ -23,32 +23,39 @@ Eco is a language aware editor. When creating a new file we need to specifiy in
...
+Eco incrementally reparses the program on every keystroke. We can see that as soon as we finish typing `</html>`, the parsing status becomes valid (![](../lib/eco/gui/accept.png)).
+
+The parse tree can be
```
Comment (ltratt) [inline tutorial/TUTORIAL.md]: I'm not sure we want to point users at this debugging view?

File: tutorial/TUTORIAL.md (hunk 1 above)
Comment (ptersilie) [inline tutorial/TUTORIAL.md:44]: Indeed it should.

File: tutorial/TUTORIAL.md (hunk 2 above)
Comment (ptersilie) [inline tutorial/TUTORIAL.md]: Good point. I'll remove it.

**Comment by ptersilie** [comment]
Fixed and squashed.

**Comment by ltratt** [comment]
Thanks!

**Pull request merged**